src/cache.ts

```diff
@@ -1,5 +1,5 @@
 import { EventBus, isObject, expiresInDays } from 'live-connect-common'
-import { WrappedStorageHandler } from './handlers/storage-handler'
+import { WrappedStorageHandler } from './handlers/storage-handler.js'
```

Comment:
Importing local `ts` files as `js` files? Wut?

Reply:
Normal node resolution mode. Esm modules want to have an explicit extension.
This can be disabled with --experimental-specifier-resolution=node, but I decided to just stick with the defaults

Reply:
This is wired and contr-intuitive. Can you at least make it consume `ts` extension? I'm not sure the navigation will work in IntelliJ as it is right now...

Reply:
See here for more details for example https://stackoverflow.com/questions/62619058/appending-js-extension-on-relative-import-statements-during-typescript-compilat.

Reply:
Can you try navigation out? It should work without any issues (as this is pretty much the 'default' setup now)